hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/BlockUtils.java

```diff
@@ -249,4 +252,64 @@ public static void removeContainerFromDB(KeyValueContainerData containerData,
           containerData.getContainerID());
     }
   }
+
+  /**
+   * Dump container KV metadata to external files.
+   * @param containerData
+   * @param conf
+   * @throws StorageContainerException
+   */
+  public static void dumpKVContainerDataToFiles(
+      KeyValueContainerData containerData,
+      ConfigurationSource conf) throws IOException {
+    try (DBHandle db = getDB(containerData, conf)) {
+      Preconditions.checkState(db.getStore()
+          instanceof DatanodeStoreSchemaThreeImpl);
+
+      DatanodeStoreSchemaThreeImpl store = (DatanodeStoreSchemaThreeImpl)
+          db.getStore();
+      File metaDir = new File(containerData.getMetadataPath());
+      try {
+        store.dumpKVContainerData(containerData.getContainerID(), metaDir);
```

Comment:
Shall we delete all existing files before dump new files?

Reply:
Could you link the document that this "SstFileWriter will reopen a fresh file each time" statement is explained?  If we provide the same dest dir, export a container, keep the dumped sst files, then export the container again, what the dest dir will look like?  For a dumped table, will there be 2 files, or just 1 file?

Reply:
https://github.com/facebook/rocksdb/blob/bcb128723578aa02f599973a2090b9c88667eb5c/table/sst_file_writer.cc#L222

Reply:
@guihecheng thanks for the info.